spec/controllers/case_management/tax_returns_controller_spec.rb

```diff
@@ -100,54 +104,243 @@
     end
   end
 
-  describe "#update_status" do
-    let(:user) { create :user, vita_partner: (create :vita_partner) }
-    let(:tax_return) { create :tax_return, status: "intake_in_progress", client: (create :client, vita_partner: user.vita_partner) }
-    let(:params) { { tax_return: { status: "review_complete_signature_requested" }, id: tax_return.id, client_id: tax_return.client } }
+  describe "#edit_status" do
+    let(:user) { create :user_with_org }
+    let(:client) { create(:client, vita_partner: user.vita_partner) }
+    let!(:intake) { create :intake, client: client }
+    let(:tax_return) { create :tax_return, client: client }
+    let(:params) { { id: tax_return.id, client_id: tax_return.client } }
 
-    it_behaves_like :a_post_action_for_authenticated_users_only, action: :update_status
+    it_behaves_like :a_get_action_for_authenticated_users_only, action: :edit_status
 
     context "as an authenticated user" do
       before { sign_in user }
 
-      it "creates a system note" do
-        expect(SystemNote).to receive(:create_status_change_note).with(user, tax_return)
-        post :update_status, params: params
+      it "returns an ok response" do
+        post :edit_status, params: params
+
+        expect(response).to be_ok
```

Comment:
Is this a POST -> GET redirect? I'm confused -- I expect a POST to either redirect to a GET (in which case, I'd expect `expect(response).to redirect_to(xyz)`), or I expect to see some purpose to the response, like `expect(assigns(:form)).to be_instance_of(...)` or something like that. Confused about this test.

Reply:
This was a typo that apparently still passed tests. Thanks for catching!